lua/entities/gmod_wire_expression2/core/iscoding.lua

```diff
@@ -0,0 +1,18 @@
+local editing = {}
+ 
+hook.Add("E2_IsEditing","E2_IsEditing_Hook",function(ply,set)
+        editing[ply] = set
+end)
+hook.Add("PlayerDisconnected","E2_IsEditing_Disconnected",function(ply)
+	editing[ply] = nil
+end)
+ 
+__e2setcost(5)
+e2function number entity:isE2Editing()
+        if !IsValid(this) then return 0 end
+        if !this:IsPlayer() then return 0 end
+       
+        //return editing[this] != nil ? 1 : 0
```

Comment:
Don't worry about that, the whole codebase just needs a Replace in Files. 

More interestingly, you can use 
return editing[this] != nil and 1 or 0
as ternary operator.